contracts/libs/0.8.x/BytecodeStorage.sol

```diff
@@ -31,6 +31,8 @@ library BytecodeStorage {
     //---------------------------------------------------------------------------------------------------------------//
     // define the offset for where the "logic bytes" end, and the "data bytes" begin
     uint256 internal constant DATA_OFFSET = 47;
+    // define the offset for where the purely-logic aspects of the the "logic bytes" end
+    uint256 internal constant DATA_OFFSET_MINUS_ADDRESS = 27;
```

Comment:
WDYT about calling this `DEPLOYER_ADDRESS_OFFSET`?

Reply:
Yeah that makes more sense, minor change so just updating now :)